force-app/main/default/lwc/pharmanetHistory/pharmanetHistory.html

```diff
@@ -28,20 +28,38 @@ <h3 slot="title">
         onchange={handleCountChange} ></lightning-combobox>
     </lightning-layout-item>
     </lightning-layout>
+    <lightning-layout>
```

Comment:
Can we fix the indent on this component?

Reply:
Added the indentation.

Reply:
I don't think you added, but it is not a show stopper so marking this feedback only as resolved.